spec/support/acceptance/examples.rb

```diff
@@ -56,7 +56,11 @@
 
   describe curl_command("https://#{host_inventory['fqdn']}/pulp/api/v3/", cacert: "#{certdir}/ca-cert.pem") do
     # Requires authentication: https://github.com/pulp/pulpcore/issues/2340
-    its(:response_code) { is_expected.to eq(403) }
+    if(ENV['BEAKER_FACTER_PULPCORE_VERSION'] == '3.21')
```

Comment:
I'd prefer to be forward compatible and check if it isn't 3.17/3.18. Also because when you run it outside of CI (where we don't set the variable) it also ends up as 3.21 because that's the default in `pulpcore::repo`.